### PR TITLE
cc-addon-admin: fix skeleton

### LINKED ISSUE #496: cc-addon-admin: skeleton story seems to be broken
The [`<cc-addon-admin>` skeleton story](https://www.clever-cloud.com/doc/clever-components/?path=/story/%F0%9F%9B%A0-addon-cc-addon-admin--skeleton) is supposed to show skeleton text inputs but it seems to be broken.

Haven't had time to check exactly why.

### PR DESCRIPTION
**Pull request by Galimede**
Fixes #496

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Changelog
 
 ## Unreleased (????-??-??)
 
-...
+* `<cc-input-text>`: fix skeleton mode
```
Comment (florian-sanders-cc) [inline CHANGELOG.md:9]: should be `<cc-addon-admin>` :wink:

**Review by pdesoyres-cc — COMMENTED** [review]
What you've done resolves the problem. But I find that playing with the component lifecycle for such small computing is overkill.
The code may be simpler if you:
* compute the `skeleton`, `name`, and `tags` information inside the `render()` method. (using local variables)
* remove the `_skeleton`, `_name`, and `_tags` properties
* remove the `update(changedProperties)` overriding.

**Comment by hsablonniere** [comment]
@pdesoyres-cc How would you handle the internal state of the _name and _tags? I think we should merge this and maybe rework it once we make progress on forms and state.

**Comment by pdesoyres-cc** [comment]
@hsablonniere you're right. I missed something during my review. What I said applies for `_skeleton`, but not for `_tags` and `_name` properties.

**Comment by hsablonniere** [comment]
@Galimede We decided to move the `_skeleton` in the render for you :wink:

**Comment by hsablonniere** [comment]
We're merging this as is. Thanks again @Galimede and thanks @pdesoyres-cc for the feedbacks.

**Pull request merged**